setup.py

```diff
@@ -62,26 +62,29 @@
 ]
 
 install_requires = [
-    'Flask-BabelEx>=0.9.2',
-    'Flask-Breadcrumbs>=0.3.0',
+    'Flask-BabelEx>=0.9.3',
+    'Flask-Breadcrumbs>=0.4.0',
     'Flask-Menu>=0.5.0',
     'Flask>=0.11.1',
-    'github3.py>=1.0.0a4',
+    'github3.py==1.0.0a4',
     'humanize>=0.5.1',
     'invenio-assets>=1.0.0b6',
-    'invenio-accounts>=1.0.0b1',
+    'invenio-accounts>=1.0.0b7',
     'invenio-celery>=1.0.0a4',
     'invenio-db>=1.0.0b3',
     'invenio-deposit>=1.0.0a8',
     'invenio-formatter[badges]>=1.0.0b3',
-    'invenio-oauth2server>=1.0.0a15',
+    'invenio-oauth2server>=1.0.0b4',
     'invenio-oauthclient>=1.0.0a12',
-    'invenio-pidstore>=1.0.0b1',
-    'invenio-records>=1.0.0b1',
+    'invenio-pidstore>=1.0.0',
+    'invenio-records>=1.0.0',
     'invenio-webhooks>=1.0.0a3',
     'mistune>=0.7.2',
     'six>=1.10.0',
-    'uritemplate.py>=0.2.0,<2.0',
+    'uritemplate.py==1.0.1',
+    'requests-oauthlib>=0.6.2,<1.2.0',
```

Comment:
Where are these requirements coming from? This can very easily cause problems with invenio-oauth2server/client down the line with incompatible versions